pandas/tests/arrays/categorical/test_analytics.py

```diff
@@ -35,31 +35,43 @@ def test_min_max(self):
         assert _min == "d"
         assert _max == "a"
 
+    @pytest.mark.parametrize("skipna", [True, False])
+    def test_min_max_with_nan(self, skipna):
+        # GH 25303
         cat = Categorical(
             [np.nan, "b", "c", np.nan], categories=["d", "c", "b", "a"], ordered=True
         )
-        _min = cat.min()
-        _max = cat.max()
-        assert np.isnan(_min)
-        assert _max == "b"
+        _min = cat.min(skipna=skipna)
+        _max = cat.max(skipna=skipna)
 
-        _min = cat.min(numeric_only=True)
-        assert _min == "c"
-        _max = cat.max(numeric_only=True)
-        assert _max == "b"
+        if skipna is False:
+            assert np.isnan(_min)
```

Comment:
use isna/notna

Reply:
`np.isnan` is a more strict / correct test in this case, since we are actually returning NaN (and not None, NA or NaT)